server/utils/adjudicationFilterHelper.ts

```diff
@@ -19,14 +19,14 @@ const error: { [key in ErrorType]: FormError } = {
 export type UiFilter = {
   fromDate: string
   toDate: string
-  status: ReportedAdjudicationStatus
+  status: ReportedAdjudicationStatus[]
```

Comment:
I think this should be
`status: ReportedAdjudicationStatus | ReportedAdjudicationStatus[]`
Explained in other comment

Reply:
So here is how the array appears in a query i just ran:

http://localhost:3000/all-completed-reports?fromDate=15%2F11%2F2022&toDate=22%2F11%2F2022&status=AWAITING_REVIEW&status=RETURNED&status=REJECTED

So the checkboxes when submitted produce essentially an array of status's
status=AWAITING_REVIEW&status=RETURNED&status=REJECTED

All the tests work using :
  status: ReportedAdjudicationStatus[]

So I dont think there is any need to make that change